judge.pl

```diff
@@ -1085,6 +1085,12 @@ sub usage
         process_request($judge->select_request);
         chdir($wd);
     }
+    if (!@{$opts{run}}) {
+        my $wd = Cwd::cwd();
+        $judge->set_def_DEs($cfg->def_DEs);
+        process_request($judge->select_request);
+        chdir($wd);
+    }
```

Comment:
Oops, my mistake.
However, there is no need for such amount of code -- see d2157866a5729e0568d07f7265ca2341ec68d78f